src/api/java/baritone/api/Settings.java

```diff
@@ -840,6 +840,17 @@ public final class Settings {
      */
     public final Setting<Integer> allowOnlyExposedOresDistance = new Setting<>(1);
 
+
+    /**
+     * Only mine blocks out of the selected area. Can't be combined with {@link #onlyMineInSelection}
+     */
+    public final Setting<Boolean> onlyMineOutOfSelection = new Setting<>(false);
+
+    /**
+     * Only mine blocks in the selected area. Can't be combined with {@link #onlyMineOutOfSelection}
+     */
+    public final Setting<Boolean> onlyMineInSelection = new Setting<>(false);
```

Comment:
I think "out of" should be "oudside" and maybe "in" should then be "inside" for consistency.
Also I think it would be better to name those `mineOnly...` so they are prefixed with the command they apply to and show up next to `mineGoalUpdateInterval`, `mineScanDroppedItems`, `mineDropLoiterDurationMSThanksLouca` and possible futures settings for `#mine` in `#set list`.